src/keboola/utils/date.py

```diff
@@ -0,0 +1,180 @@
+import dateparser
+import math
+import pytz
+from _datetime import timedelta
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from typing import Tuple, Generator, Union, Dict, List
+
+date_tuple = Union[Tuple[datetime, datetime], Tuple[str, str]]
+date_gen = Generator[Dict[str, datetime], None, None]
+date_chunk = Union[date_gen, List]
+
+
+def get_date_period_converted(period_from: str, period_to: str, strformat: str = None) -> date_tuple:
```

Comment:
I hate myself for coming up with such a stupid name, what does it even mean `date_period_converted`? maybe parse_datetime_interval or get_datetime_interval_parsed or something? I don't know really but this name feels dumb.

Reply:
I like `parse_datetime_interval()`. Renamed it to that.